sycl/unittests/kernel-and-program/DeviceInfo.cpp

```diff
@@ -85,3 +94,35 @@ TEST_F(DeviceInfoTest, GetDeviceUUID) {
       << "Expect device UUID to be "
       << "of 16 bytes";
 }
+
+TEST_F(DeviceInfoTest, BuiltInKernelIDs) {
+  if (Plt.is_host()) {
+    return;
+  }
+
+  context Ctx{Plt.get_devices()[0]};
+  TestContext.reset(new TestCtx(Ctx));
+  TestContext->BuiltInKernels = "Kernel0;Kernel1;Kernel2";
+
+  device Dev = Ctx.get_devices()[0];
+
+  auto ids = Dev.get_info<info::device::built_in_kernel_ids>();
+
+  EXPECT_EQ(ids.size(), 3u);
+  EXPECT_EQ(ids[0].get_name(), std::string{"Kernel0"});
```

Comment:
Nit. Strictly speaking you don't need `std::string` (though I'm not opposed it)
```suggestion
  EXPECT_EQ(ids[0].get_name(), "Kernel0");
```

Reply:
`get_name` returns `const char *`. Changed to `EXPECT_STREQ`.